cloudant-client/src/main/java/com/cloudant/client/api/views/SettableViewParameters.java

```diff
@@ -52,6 +52,40 @@
      */
     String STALE_UPDATE_AFTER = "update_after";
 
+    /**
+     * Constant for the value "false" for use with
+     * {@link com.cloudant.client.internal.views.CommonViewRequestBuilder#update(String)}
```

Comment:
This should link to the API method in `SettableViewParameters` I think rather than the internal impl.

Reply:
Thanks for spotting, I think they were auto-completes. Fixed in 73bc905